django/forms/boundfield.py

```diff
@@ -278,6 +278,13 @@ def build_widget_attrs(self, attrs, widget=None):
                 attrs["required"] = True
         if self.field.disabled:
             attrs["disabled"] = True
+        id_for_label = self.id_for_label
+        if self.field.help_text and id_for_label:
+            helptext_id = '%s_helptext' % id_for_label
```

Comment:
Please format changed files using the [black](https://black.readthedocs.io/en/stable/) auto-formatter.